### PR TITLE
Styles Change For Auctions Page

### LINKED ISSUE #297: [UI Revamp] Auction listing
### Expectation

* Without changing or with minimum change on functionality, closely align the UI as per [Figma](https://www.figma.com/file/9t1Zr4PGQL4UWSIpeh4RMY/RDS?node-id=0%3A1&t=RLiIlpzU3voEjvTW-3)
Screenshot 2022-12-11 at 4 05 41 PM

### PR DESCRIPTION
**Pull request by bajajcodes**
## What's The Change
- Layout of Auction page header and cards section
- Create New Auction button Style
- Added SVG file `white-plus-icon` to use icon and text for Create New Auction button 
- Card Component layout
- Cards are laid out to left, as per the new design. For Refference Check [Figma](https://www.figma.com/file/9t1Zr4PGQL4UWSIpeh4RMY/RDS?node-id=2%3A270&t=g85H2Qm5N4KIzVWW-0)

## Before The Change

https://user-images.githubusercontent.com/29247011/219954158-69d6a5cf-6c8f-4d19-a5c2-888c0b1c706d.mp4

## After The Change

https://user-images.githubusercontent.com/29247011/219953178-66237889-5143-4d94-a3a1-72ebbd6205b6.mp4

## Checklist
- [X] Responsive
- [ ] Without changing or with minimum change on functionality, closely align the UI as per [Figma](https://www.figma.com/file/9t1Zr4PGQL4UWSIpeh4RMY/RDS?node-id=0%3A1&t=RLiIlpzU3voEjvTW-3) **REASON: number of changes are greater in number than expected**
- [ ] Ease To Extend **REASON: No, missing utility classes. Required to create these utility classes** [Will Disucss With AO]

#### Meta Info
- Closes [ [UI Revamp] Auction listing #297 ](https://github.com/Real-Dev-Squad/website-crypto/issues/297)
- Using more of redundant styles and features for layout(grid, flex), centering(justify and align) and font(weight, size), will refactor post discussion with AO.

### CODE REVIEW

File: components/Auction/Auction.module.css [hunk 1]
```diff
@@ -1,167 +1,158 @@
...
-.mainContainer {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  margin-bottom: 100px;
+.auctionButton {
+  max-width: 13.125rem;
+  width: 100%;
+  height: 3.125rem;
+  padding: 2px 0.625rem;
```
Comment (Muralidhar22) [inline components/Auction/Auction.module.css:23]: why `px` and why it's combined with rem

File: components/Auction/index.js [hunk 2]
```diff
@@ -113,34 +113,53 @@ const HandleAuctions = () => {
...
+                <div
+                  className={styles.biddersImg}
+                  key={bidder}
+                  data-columns={getColumns(bidders.length)}
+                  title={bidder}
+                >
+                  <img
+                    className={styles.profilePhoto}
+                    src={`${BASE_IMAGE_URL}/${bidder}/img.png`}
+                    onError={brokenImageHandler}
```
Comment (Muralidhar22) [inline components/Auction/index.js:155]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#browser_compatibility
can you please check this it seems like being deprecated
image

File: components/Auction/Auction.module.css [hunk 3]
```diff
@@ -1,167 +1,158 @@
...
-  font-weight: 550;
-  height: 50px;
-  font-size: 1.2rem;
-  width: 14rem;
-  border: none;
+.auctionButtonContainer {
   text-align: center;
-  padding: 2px 10px;
-  border-radius: 10px;
-  background-color: #101f92;
```
Comment (sahsisunny) [inline components/Auction/Auction.module.css:14]: can you add the variable of the colors ?

**Review by sahsisunny — CHANGES_REQUESTED** [review]

File: pages/auction/index.js [hunk 4]
```diff
@@ -1,20 +1,26 @@
...
         <meta name="viewport" content="initial-scale=1.0, width=device-width" />
       </Head>
 
       <Link href="/auction/new">
-        <div className={styles.newAuctionBtnContainer}>
-          <button className={styles.newAuctionBtn}>Create New Auction</button>
+        <div className={styles.auctionButtonContainer}>
+          <button className={styles.auctionButton}>
+            <div className={styles.auctionButtonContentWrapper}>
+              <Image width="18" height="20" src="/assets/white-plus-icon.svg" />
```
Comment (manish591) [inline pages/auction/index.js]: Please add image `alt`.

File: components/Auction/index.js (hunk 2 above)
Comment (manish591) [inline components/Auction/index.js:156]: Please add image `alt`.

File: components/Auction/Auction.module.css [hunk 5]
```diff
@@ -1,167 +1,158 @@
...
-  flex-direction: column;
-  align-items: center;
-  margin-bottom: 100px;
+.auctionButton {
+  max-width: 13.125rem;
+  width: 100%;
+  height: 3.125rem;
+  padding: 2px 0.625rem;
+  color: white;
+  background: #0094b6;
```
Comment (samyakshah3008) [inline components/Auction/Auction.module.css:25]: Can we use CSS variables?

**Review by samyakshah3008 — DISMISSED** [review]
Great work, looks good to merge!

File: components/Auction/Auction.module.css (hunk 1 above)
Comment (bajajcodes) [reply components/Auction/Auction.module.css:23]: If values are less than 8 then I am not using `rem` as unit for them.

File: components/Auction/Auction.module.css (hunk 5 above)
Comment (bajajcodes) [reply components/Auction/Auction.module.css:25]: Add css variables, for reusable colors.

File: components/Auction/Auction.module.css (hunk 1 above)
Comment (bajajcodes) [reply components/Auction/Auction.module.css:23]: Using px as unit for values less than 8

Comment (Muralidhar22) [reply components/Auction/Auction.module.css:23]: just want to know what's the benefit of doing like this, imo for padding it's better to use `rem`

**Review by manish591 — DISMISSED** [review]

File: components/Auction/Auction.module.css [hunk 6]
```diff
@@ -1,167 +1,167 @@
...
+  margin-bottom: 6.25rem;
 }
 
-.auctionSeller {
-  width: 100px;
+.autionsPageTitle {
+  font-size: 1.5rem;
+  font-weight: 600;
+  line-height: 2rem;
+  color: #352e5b;
```
Comment (sahsisunny) [inline components/Auction/Auction.module.css:51]: Forgot to use variable

File: components/Auction/Auction.module.css [hunk 7]
```diff
@@ -1,167 +1,167 @@
...
+  display: grid;
+  grid-template-columns: 5rem 1fr auto;
+  gap: 2rem;
 }
 
-.currentBidPrice {
-  font-weight: 400;
+.auctionContainer:hover {
+  box-shadow: 0px 3px 6px rgba(75, 81, 91, 0.15),
+    0px 1px 3px rgba(0, 0, 0, 0.15);
```
Comment (sahsisunny) [inline components/Auction/Auction.module.css:72]: can you also create the variable for rgba ?

File: components/Auction/Auction.module.css [hunk 8]
```diff
@@ -1,167 +1,167 @@
...
+.gemImage {
+  margin-top: 1.125rem;
 }
 
-.biddersImg[data-columns='3'] {
-  width: calc(160px / 3);
+.currentBidPrice {
+  font-weight: 900;
+  font-size: 1.125rem;
+  color: #e30062;
```
Comment (sahsisunny) [inline components/Auction/Auction.module.css:118]: you forgot to use variable for color

File: components/Auction/Auction.module.css [hunk 9]
```diff
@@ -1,167 +1,167 @@
...
-  width: calc(160px / 2);
+.gemImage {
+  margin-top: 1.125rem;
 }
 
-.biddersImg[data-columns='3'] {
-  width: calc(160px / 3);
+.currentBidPrice {
+  font-weight: 900;
+  font-size: 1.125rem;
```
Comment (sahsisunny) [inline components/Auction/Auction.module.css]:
```suggestion
  font: 900 1.125rem;
```

File: components/Auction/Auction.module.css [hunk 10]
```diff
@@ -1,167 +1,167 @@
...
+  display: flex;
+  gap: 4px;
 }
 
-.biddersImg {
-  margin-left: 10px;
+.itemInfo,
+.currentStatus {
+  font-weight: 700;
+  font-size: 1.125rem;
```
Comment (sahsisunny) [inline components/Auction/Auction.module.css]: can we use shorthand for font ?

File: components/Auction/Auction.module.css (hunk 1 above)
Comment (bajajcodes) [reply components/Auction/Auction.module.css:23]: This is not about benefit, but more of personal prefference.

File: components/Auction/Auction.module.css (hunk 6 above)
Comment (bajajcodes) [reply components/Auction/Auction.module.css:51]: Hey, @sahsisunny I intentionally didn't use colors variable, 
as this and other color values are used on time only. 

Post connecting with AO and figuring out common values, 
I will raise ANother PR to use variables for colors.

File: components/Auction/Auction.module.css (hunk 7 above)
Comment (bajajcodes) [reply components/Auction/Auction.module.css:72]: Check this comment: https://github.com/Real-Dev-Squad/website-crypto/pull/306#discussion_r1114160275

File: components/Auction/Auction.module.css (hunk 8 above)
Comment (bajajcodes) [reply components/Auction/Auction.module.css:118]: Check this comment: https://github.com/Real-Dev-Squad/website-crypto/pull/306#discussion_r1114160275

File: components/Auction/Auction.module.css [hunk 11]
```diff
@@ -1,167 +1,162 @@
...
-  width: calc(160px / 5);
+.bidOptions {
+  display: flex;
+  flex-direction: column;
 }
 
-.biddersImg[data-columns='6'] {
-  width: calc(160px / 6);
+.inputBid {
+  border: 0.5px solid #ccc;
```
Comment (Muralidhar22) [inline components/Auction/Auction.module.css:133]: can you please use hex value, it will be consistent

**Review by MehulKChaudhari — APPROVED** [review]

File: components/Auction/Auction.module.css (hunk 11 above)
Comment (bajajcodes) [reply components/Auction/Auction.module.css:133]: Done.

**Review by sahsisunny — APPROVED** [review]

**Pull request merged**